assets/section-main-product.css

```diff
@@ -595,15 +604,44 @@ a.product__text {
   .grid__item.product__media-item--full {
     width: 100%;
   }
+
+  .product--columns .product__media-item:not(.product__media-item--variant) {
+    max-width: calc(50% - var(--grid-desktop-horizontal-spacing) / 2);
+  }
+
+  .product--large.product--columns .product__media-item--full .deferred-media__poster-button {
+    height: 5rem;
+    width: 5rem;
+  }
+
+  .product--medium.product--columns .product__media-item--full .deferred-media__poster-button {
+    height: 4.2rem;
+    width: 4.2rem;
+  }
+
+  .product--medium.product--columns .product__media-item--full .deferred-media__poster-button .icon {
+    width: 1.8rem;
+    height: 1.8rem;
+  }
+
+  .product--small.product--columns .product__media-item--full .deferred-media__poster-button {
+    height: 3.6rem;
+    width: 3.6rem;
+  }
+
+  .product--small.product--columns .product__media-item--full .deferred-media__poster-button .icon {
+    width: 1.6rem;
+    height: 1.6rem;
+  }
```

Comment:
I do ❤️ this suggestion as well.

However, I'd be more inclined to revisit this in the future + add it to the list of many CSS improvements that can be made to our stylesheets (and other files too, not just CSS). If you both agree then I'm also more than happy to keep this as-is for now.

I'll also add that ever since joining the team + ramping up my Dawn codebase knowledge through PR reviews and PDP work, and with the addition of the Prettier plugin, the `section-main-product.css` was one of first assets that came to mind that I'd very much like to refactor/improve as it contains a lot of areas of improvement 👍🏼

Reply:
> section-main-product.css was one of first assets that came to mind

No doubt. I was really hoping to have a chance to clean up the main product section before adding more complexity to it.. The CSS definitely has some improvements we can make as is, but it may well be that some additional abstraction and reorganization of the main product section in general would go the longest way in improving the readability and extensibility of this and related files.

> If you both agree then I'm also more than happy to keep this as-is for now.

I mean if you're sold on it, by all means include it here. I was just expressing you don't _need_ to feel inclined to do it if you didn't feel strongly yourself. I love using lots of css variables, but I try to consider the balance between the complexity/abstraction that additional variables will add versus how many instances/iterations they will help to streamline currently and in foreseeable future work. This particular instance in my mind reads as "mostly worthwhile" 🙂

Reply:
Thanks for sharing your perspective, Ken! 😄 🥯 - I'm completely aligned with everything.

> I was really hoping to have a chance to clean up the main product section before adding more complexity to it..

I would love to hear your fresh take (_added complexity_) on the recent desktop media position PR. But I think this was one of those platform limitation scenarios. Still would've loved your perspective though.

>I mean if you're sold on it, by all means include it here.

At the end of the day, I'd still prefer adding the improvements [mentioned here](https://github.com/Shopify/dawn/pull/1766#discussion_r944871790) in a separate PR and I'm more than happy to create an issue for it (eg "Refactor PDP etc."). I would actually love if I could also create the PR for it and also get your perspective/review + see if we can tackle other things / tasks related to cleaning up the main product section (as you've mentioned) -- keeping other team tasks and priorities in mind of course. 

If you're aligned with that I'll create the issue 👍🏼

Reply:
Thanks for the great suggestion, guys! 

I've created an issue, please feel free to add to the list 👍🏼 

- https://github.com/Shopify/dawn/issues/1931